ci/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM openjdk:11-slim-buster
+
+# Install Python and other dependencies
+ARG VIRTUAL_ENV=/opt/python3
+
+RUN apt-get update \
+ && apt-get install -y --no-install-recommends \
```

Comment:
nit: I would keep `&& \` at the end of line for consistency with other dockerfiles (other repos, see original one)

Reply:
Done.